docs/source/conf.py

```diff
@@ -85,25 +85,37 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-import guzzle_sphinx_theme
 
-html_theme_path = guzzle_sphinx_theme.html_theme_path()
-html_theme = "guzzle_sphinx_theme"
+html_theme = "furo"
 html_favicon = str(Path("_static/favicon.ico"))
 
-# There's a standing issue with Sphinx's new-style sidebars.  This is a
-# workaround.  Taken from
-# https://github.com/guzzle/guzzle_sphinx_theme/issues/33#issuecomment-637081826
-html_sidebars = {"**": ["logo-text.html", "globaltoc.html", "searchbox.html"]}
-
-# Register the theme as an extension to generate a sitemap.xml
-extensions.append("guzzle_sphinx_theme")
-
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
 
+html_theme_options = {
+    'light_logo': 'manim-logo-sidebar.svg',
+    'dark_logo': 'manim-logo-sidebar-dark.svg',
+    'light_css_variables': {
+        'color-content-foreground': '#413c3c',
+        'color-background-primary': '#f1ece9',
+        'color-background-border': '#ded6d4',
+        'color-sidebar-background': '#ece6e2',
+        'color-brand-content': '#2196f3',
+        'color-brand-primary': '#525893',
+    },
+    'dark_css_variables': {
+        'color-content-foreground': '#ffffffd9',
+        'color-background-primary': '#131416',
+        'color-background-border': '#303335',
+        'color-sidebar-background': '#1a1c1e',
+        'color-brand-content': '#2196f3',
+        'color-brand-primary': '#87c2a5',
+    }
+}
+html_title = "Manim Community"
```

Comment:
Maybe change it to like `Manim Documentation` like how it was previously.

Reply:
If it's possible using an f-string, I would like the version number to be in the title as well similar to how Python does it:
https://docs.python.org/3/

Reply:
Suggestion: 
![image](https://user-images.githubusercontent.com/11851593/114110768-8e23c980-98d8-11eb-89a8-3b945052f406.png)

Whenever I announce a new version, I announce it as "Manim Community vX.Y.Z", and I actually think it makes more sense to underline that this page concerns the community edition in version X.Y.Z, than to point out that this is the documentation.

Pushed via 751accea.